tjreports/site/controllers/reports.json.php

```diff
@@ -131,4 +131,37 @@ public function deleteQuery()
 			echo new JResponseJson($e);
 		}
 	}
+
```

Comment:
Shouldn't this be in an API resource instead of task ?

Reply:
This is written to take the reports those supports google data connector. Do we really need rest API for this?